src/PrestaShopBundle/Form/Admin/Sell/Address/CustomerAddressType.php

```diff
@@ -87,12 +98,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $data = $builder->getData();
         $countryId = 0 !== $data['id_country'] ? $data['id_country'] : $this->contextCountryId;
+        $genericInvalidCharsMessage = $this->translator->trans(
+            'Invalid characters: %s',
```

Comment:
Why not keep the string as written elsewhere in the software: _Invalid characters:_ in _Admin.Notifications.Info_?

Reply:
Following what decision is made, it will have to be harmonized in all occurrences in the PR.

Reply:
@matks is the "_%s_" of this string necessary (for listing the characters for instance)? If not, we could keep "_Invalid characters:_", which already exists, like Louise suggested!